mantidproject/mantid
# Migrate Current Set of Release Notes for v3.7.0 to rst

The current set of release notes on the [wiki](http://www.mantidproject.org/Release_Notes_3.7) have been ported to rst following #15486. 

**To test:**
- Build the documentation and check there are no warnings concerning the new pages
- Check the pages locally to see that the formatting is correct
- Check that the note put on the main page regarding "being under construction" makes sense

No issue number

"Does not need to be in the release notes."

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
